### PR TITLE
Filter netdata incoming connection

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Manage iptables rules for netdata.
DROP packet that not coming from the admin to netdata

# Impacts
iptables rules

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* Protect netdata from incoming connection

### CODE REVIEW

File: lib/pf/iptables.pm [hunk 1]
```diff
@@ -140,6 +141,17 @@ sub iptables_generate {
...
+    $tags{'netdata'} = "-A input-management-if --protocol tcp --match tcp -s 127.0.0.1 --dport 19999 --jump ACCEPT\n";
+
+    if ($cluster_enabled) {
+        push my @mgmt_backend, map { $_->{management_ip} } pf::cluster::config_enabled_servers();
+
+        foreach my $mgmt_back (uniq(@mgmt_backend)) {
+            $tags{'netdata'} .= "-A input-management-if --protocol tcp --match tcp -s $mgmt_back --dport 19999 --jump ACCEPT\n";
+        }
+    }
+    $tags{'netdata'} .= "-A input-management-if --protocol tcp --match tcp --dport 19999 --jump DROP\n";
```
Comment (JeGoi) [inline lib/pf/iptables.pm:153]: Here the idea is to drop everything that is not related to a management IP or localhost, correct?

**Review by JeGoi — APPROVED** [review]
Just one question to clarified the drop, but not an issue.
Then it is fine for me.

File: lib/pf/iptables.pm (hunk 1 above)
Comment (fdurand) [reply lib/pf/iptables.pm:153]: Yes, since we reverse proxy the netdata interface, only ip addresses from the management and localhost needs to have access.

**Comment by fdurand** [comment]
BUMP
